### PR TITLE
deploy using tiup: align three PRs

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
  - https://github.com/pingcap/docs-cn/pull/7476
  - https://github.com/pingcap/docs-cn/pull/8363
  - https://github.com/pingcap/docs-cn/pull/9399
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -244,106 +268,96 @@ alertmanager_servers:
...
-- [Hybrid deployment topology](/hybrid-deployment-topology.md)
-
-    This is to deploy multiple instances on a single machine. You need to add extra configurations for the directory, port, resource ratio, and label.
-
-- [Geo-distributed deployment topology](/geo-distributed-deployment-topology.md)
-
-    This topology takes the typical architecture of three data centers in two cities as an example. It introduces the geo-distributed deployment architecture and the key configuration that requires attention.
+| Application | Configuration task | Configuration file template | Topology description |
+| :-- | :-- | :-- | :-- |
+| OLTP | [Deploy minimal topology](/minimal-deployment-topology.md) | <ul><li>[Simple minimal configuration template](/config-templates/simple-mini.yaml) </li> <li> [Full minimal configuration template](/config-templates/complex-mini.yaml) | This is the basic cluster topology, including tidb-server, tikv-server, and pd-server. |
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:275]:
```suggestion
| OLTP | [Deploy minimal topology](/minimal-deployment-topology.md) | <ul><li>[Simple minimal configuration template](/config-templates/simple-mini.yaml) </li> <li> [Full minimal configuration template](/config-templates/complex-mini.yaml) </li> </ul>| This is the basic cluster topology, including tidb-server, tikv-server, and pd-server. |
```

File: production-deployment-using-tiup.md [hunk 2]
```diff
@@ -195,28 +202,45 @@ sh tidb-community-server-${version}-linux-amd64/local_install.sh && \
...
 ```shell
 tiup cluster template > topology.yaml
 ```
 
-> **Note:**
->
-> For the hybrid deployment scenarios, you can also execute `tiup cluster template --full > topology.yaml` to create the recommended topology template. For the geo-distributed deployment scenarios, you can execute `tiup cluster template --multi-dc > topology.yaml` to create the recommended topology template.
+In the following two common scenarios, you can generate recommended topology templates by running commands:
+
+- Hybrid deployment: Multiple instances are deployed on a single machine. For details, see [Hybrid Deployment Topology](/hybrid-deployment-topology.md).
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:224]:
```suggestion
- For hybrid deployment: Multiple instances are deployed on a single machine. For details, see [Hybrid Deployment Topology](/hybrid-deployment-topology.md).
```

File: production-deployment-using-tiup.md [hunk 3]
```diff
@@ -195,28 +202,45 @@ sh tidb-community-server-${version}-linux-amd64/local_install.sh && \
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster template --full > topology.yaml
+    ```
+
+- Geo-distributed deployment: TiDB clusters are deployed in geographically distributed data centers. Fordetails, see [Geo-Distributed Deployment Topology](/geo-distributed-deployment-topology.md).
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:232]:
```suggestion
- For geo-distributed deployment: TiDB clusters are deployed in geographically distributed data centers. For details, see [Geo-Distributed Deployment Topology](/geo-distributed-deployment-topology.md).
```

File: production-deployment-using-tiup.md [hunk 4]
```diff
@@ -195,28 +202,45 @@ sh tidb-community-server-${version}-linux-amd64/local_install.sh && \
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster template --multi-dc > topology.yaml
+    ```
 
 Execute `vi topology.yaml` to see the configuration file content:
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:240]:
```suggestion
Run `vi topology.yaml` to see the configuration file content:
```

File: production-deployment-using-tiup.md [hunk 5]
```diff
@@ -244,106 +268,96 @@ alertmanager_servers:
...
+3. Deploy a TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster deploy tidb-test v6.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+    ```
 
-In the above command:
+In the preceding command:
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:338]:
```suggestion
In the `tiup cluster deploy` command above:
```

File: production-deployment-using-tiup.md [hunk 6]
```diff
@@ -244,106 +268,96 @@ alertmanager_servers:
...
 
-- The name of the deployed TiDB cluster is `tidb-test`.
-- You can see the latest supported versions by running `tiup list tidb`. This document takes `v6.0.0` as an example.
-- The initialization configuration file is `topology.yaml`.
-- `--user root`: Log in to the target machine through the `root` key to complete the cluster deployment, or you can use other users with `ssh` and `sudo` privileges to complete the deployment.
-- `[-i]` and `[-p]`: optional. If you have configured login to the target machine without password, these parameters are not required. If not, choose one of the two parameters. `[-i]` is the private key of the `root` user (or other users specified by `--user`) that has access to the target machine. `[-p]` is used to input the user password interactively.
-- If you need to specify the user group name to be created on the target machine, see [this example](https://github.com/pingcap/tiup/blob/master/embed/examples/cluster/topology.example.yaml#L7).
+- `tidb-test` is the name of the deployed TiDB cluster.
+- `v6.0.0` is the version of the deployed TiDB cluster. You can see the latest supported versions by running `tiup list tidb`.
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:341]:
```suggestion
- `v6.0.0` is the version of the TiDB cluster to be deployed. You can see the latest supported versions by running `tiup list tidb`.
```

File: scale-tidb-using-tiup.md [hunk 7]
```diff
@@ -24,19 +24,19 @@ For example, if the original topology of the cluster is as follows:
 
 ## Scale out a TiDB/PD/TiKV cluster
 
-If you want to add a TiDB node to the `10.0.1.5` host, take the following steps.
+Assume that you want to add a TiDB node to the `10.0.1.5` host. Take the following steps.
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:27]: 感觉修改前的版本更好

File: scale-tidb-using-tiup.md [hunk 8]
```diff
@@ -91,29 +91,41 @@ If you want to add a TiDB node to the `10.0.1.5` host, take the following steps.
...
+
+        ```shell
+        tiup cluster scale-out <cluster-name> scale-out.yaml [-p] [-i /home/root/.ssh/gcp_rsa]
+        ```
+
+    In the preceding commands:
 
-    If you see the `Scaled cluster <cluster-name> out successfully`, the scale-out operation is successfully completed.
+    - `scale-out.yaml` is the scale-out configuration file.
+    - `--user root` indicates logging in to the target machine as the root user to complete the cluster scale out. The root user should have`ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:125]: ditto

File: scale-tidb-using-tiup.md [hunk 9]
```diff
@@ -137,14 +149,14 @@ After the scale-out, the cluster topology is as follows:
 
 ## Scale out a TiFlash cluster
 
-If you want to add a TiFlash node to the `10.0.1.4` host, take the following steps.
+Assume that you want to add a TiFlash node to the `10.0.1.4` host. Take the following steps.
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:152]: ditto, 个人倾向于原来的

File: scale-tidb-using-tiup.md [hunk 10]
```diff
@@ -324,29 +333,29 @@ The current topology is as follows:
...
-    alter table <db-name>.<table-name> set tiflash replica 0;
+    ALTER TABLE <db-name>.<table-name> SET tiflash replica 0;
     ```
 
 2. Wait for the TiFlash replicas of the related tables to be deleted. [Check the table replication progress](/tiflash/use-tiflash.md#check-replication-progress) and the replicas are deleted if the replication information of the related tables is not found.
 
 ### 2. Perform the scale-in operation
 
-Next, perform the scale-in operation with one of the following solutions.
+perform the scale-in operation with one of the following solutions.
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:354]:
```suggestion
Perform the scale-in operation with one of the following solutions.
```

**Review by TomShawn — APPROVED** [review]
great work!

File: scale-tidb-using-tiup.md (hunk 8 above)
Comment (shichun-0415) [inline scale-tidb-using-tiup.md:125]:
```suggestion
    - `--user root` indicates logging in to the target machine as the `root` user to complete the cluster scale out. The `root` user is expected to have `ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.
```

Comment (shichun-0415) [inline scale-tidb-using-tiup.md:125]:
```suggestion
    - `--user root` indicates logging in to the target machine as the `root` user to complete the cluster scale out. The `root` user is expected to have `ssh` and `sudo` privileges to the target machine. Alternatively, you can use other users with `ssh` and `sudo` privileges to complete the deployment.
```

**Review by qqqdan — APPROVED** [review]

**Pull request merged**